### PR TITLE
System read code simplification

### PR DESCRIPTION
**Pull request by timja**
See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

### Proposed changelog entries

* No need
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy [hunk 1]
```diff
@@ -17,9 +17,7 @@ l.layout(permission:app.SYSTEM_READ, title:my.displayName, cssclass:request.getP
             l.icon(class: 'icon-secure icon-xlg')
             text(my.displayName)
         }
-        if (!h.hasPermission(app.ADMINISTER)) {
-            set("readOnlyMode", "true")
-        }
+        set("readOnlyMode", !h.hasPermission(app.ADMINISTER))
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy:20]: Seems like an oversight to use `Functions` without need.

File: core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy [hunk 2]
```diff
@@ -7,18 +7,14 @@ def f=namespace(lib.FormTagLib)
...
-}
-
 l.layout(permission:app.SYSTEM_READ, title:my.displayName) {
     l.side_panel {
         l.tasks {
             l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
             l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"))
         }
     }
+    set("readOnlyMode", !h.hasPermission(app.ADMINISTER))
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index.groovy:17]: As above.

**Review by daniel-beck — APPROVED** [review]
Assuming it has been manually tested, I don't think we have autotests? Unsure.

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy (hunk 1 above)
Comment (timja) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy:20]: are you saying `Functions` should be used here or h?

**Comment by timja** [comment]
> Assuming it has been manually tested, I don't think we have autotests? Unsure.

I don't think it's covered (possibly the one on /configure is) but yes I've manually checked all the changes

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy (hunk 1 above)
Comment (daniel-beck) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy:20]: `app.hasPermission` as in some other of these changes.

Comment (timja) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy]: done

**Review by jvz — APPROVED** [review]

File: core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy (hunk 1 above)
Comment (daniel-beck) [reply core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy]: (Or `Jenkins.get()` which is the nicer, type-safer variant for Groovy views, but really the improvement is marginal. Not worth redoing, just a recommendation for the future.)

**Review by daniel-beck — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Comment by res0nance** [comment]
Merging in 24h if no negative feedback

**Pull request merged**